src/assisted_test_infra/test_infra/controllers/minikube_controller.py

```diff
@@ -0,0 +1,289 @@
+import logging
+import time
+from enum import Enum
+
+import kubernetes.client
+import kubernetes.config
+import kubernetes.dynamic
+import waiting
+
+from tests.global_variables.default_variables import DefaultVariables
+
+global_variables = DefaultVariables()
```

Comment:
Remove this from here. There is only single instance of DefaultVariables.?
You can pass the vars you need from base_test